pkg/util/mount/mount_linux.go

```diff
@@ -282,7 +282,7 @@ func readProcMounts(mountFilePath string, out *[]MountPoint) (uint32, error) {
 }
 
 func readProcMountsFrom(file io.Reader, out *[]MountPoint) (uint32, error) {
-	hash := adler32.New()
+	hash := fnv.New32a()
```

Comment:
I guess kubelet is using this utility. Mind explaining what's the impact of this change?